Correct spelling and wording (#144)
homieiot/convention: convention.md

As soon as a device starts to publish any Homie related topic,
it MUST finish with all topics within a timeframe of 500ms.

> Controllers should assume an empty string for topics values not received within this timeframe.

Reviewer: ```suggestion
Controllers should assume the default for topic values not received within this timeframe.
```

Reviewer: Up for discussion. Should we distinguish between mandatory and optional attributes?

## Extensions

This convention only covers discoverability of devices and its capabilities.
The aim is to have standardized MQTT topics for all kind of complex scenarios.
A Homie device may therefore support extensions, defined in separate documents.

> Every extension is identified by a unique id and will be linked from the section.

Reviewer: ```suggestion
Every extension is identified by a unique ID and will be linked from the section.
```

> The id consists of the reverse domain name and a freely choosen prefix. The domain and prefix `homie` is reserved.

Reviewer: ```suggestion
The ID consists of the reverse domain name and a freely chosen prefix.
The domain and prefix `homie` is reserved.
```

> For example an organisation `example.org` wants to add a feature `our-feature`. They would choose the extension id `org.example.our-feature`. 

Reviewer: ```suggestion
For example, an organization `example.org` wanting to add a feature `our-feature` would choose the extension ID `org.example.our-feature`. 
```

## Extensions

This convention only covers discoverability of devices and its capabilities.
The aim is to have standardized MQTT topics for all kind of complex scenarios.
A Homie device may therefore support extensions, defined in separate documents.
Every extension is identified by a unique ID and will be linked from the section.

> The ID consists of the reverse domain name and a freely chosen prefix.

Reviewer 2: I know I wrote the original, but noticed it just now.
Isn't this a postfix not a prefix ?

Author: suffix is the word you are looking for ^^

This convention only covers discoverability of devices and its capabilities.
The aim is to have standardized MQTT topics for all kind of complex scenarios.
A Homie device may therefore support extensions, defined in separate documents.
Every extension is identified by a unique ID and will be linked from this section.

The ID consists of the reverse domain name and a freely chosen suffix.

> The domain and suffix `homie` is reserved.

Reviewer 2: Well `homie` was really a prefix, isn't it ?
Only the first time it should be suffix.

Reviewer: See #147
